src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+management.endpoints.enabled-by-default=false
+management.endpoints.web.base-path=/account-validator
+management.endpoints.web.path-mapping.health=/healthcheck
+management.endpoint.health.show-details=never
+management.endpoint.health.enabled=true
+
+chs.kafka.api.endpoint=${CHS_KAFKA_API_URL:localhost}
+chs.kafka.api.key=${CHS_API_KEY:chsApiKey}
+
+spring.jackson.default-property-inclusion=NON_NULL
+
+server.port=54774
```

Comment:
How was this port chosen? Should we go for something around 18000 like most other services. See https://github.com/companieshouse/chs-deployer/blob/feature/marathon_lb_routes/appconfig.yml

Reply:
not yet, it needs to be defined

Reply:
server.port=18999
and it needs to be updated